amlip_cpp/src/cpp/node/collaborative_learning/ModelManagerReceiverNode.cpp

```diff
@@ -179,7 +195,7 @@ eprosima::fastdds::dds::DataReaderQos ModelManagerReceiverNode::default_statisti
     qos.durability().kind = eprosima::fastdds::dds::DurabilityQosPolicyKind::TRANSIENT_LOCAL_DURABILITY_QOS;
     qos.reliability().kind = eprosima::fastdds::dds::ReliabilityQosPolicyKind::RELIABLE_RELIABILITY_QOS;
     qos.history().kind = eprosima::fastdds::dds::HistoryQosPolicyKind::KEEP_LAST_HISTORY_QOS;
-    qos.history().depth = 1;
+    qos.history().depth = 10;
```

Comment:
Please add a TODO comment to revisit this. Probably in the future we will need to pass qos to the constructor.